### DISCUSSION TITLE
Change animation speed?

### DISCUSSION THREAD

**Discussion by Rmano** [Schneegans/Burn-My-Windows#409] [Q&A] [answered]
Hi!

First of all, this is one of the funniest extension I have ever seen. Thanks, @Schneegans !

A quick question: it is possible to speed up the animations? I quite like them, but I'd like to have them played at more or less double speed.

Romano

**Comment by Schneegans** [comment, marked as answer]
Thanks! Of course you can. Just open the settings of your effect and change the animation time slider!

![image](https://github.com/Schneegans/Burn-My-Windows/assets/829942/5a7cf40d-bbbd-4f67-b9c3-57559899d94e)

**Comment by Rmano** [comment]
How could I missed it... Thanks for pointing it out (and happy new year, btw).

Anyway, just not to seem too dumb, I'll turn this into a suggestion: what about a global modifier to speed up/down every module together?
